config/controller.yaml

```diff
@@ -67,7 +67,8 @@ spec:
           "-build-gcs-fetcher-image", "github.com/tektoncd/pipeline/vendor/github.com/GoogleCloudPlatform/cloud-builders/gcs-fetcher/cmd/gcs-fetcher",
 
           # These images are pulled from Dockerhub, by digest, as of May 19, 2020.
-          "-nop-image", "tianon/true@sha256:183cb5fd54142948ad88cc986a217465cfe8786cfdd89b1ed1fc49825da413a7",
```

Comment:
This is interesting to me -- this digest can't have worked previously, because this is the digest of the image config, not an image manifest (unless Docker somehow does something clever when given a config object digest? but I don't think it does):

```json
digest: sha256:009cce421096698832595ce039aa13fa44327d96beedb84282a69d3dbcf5a81b
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
   "config": {
      "mediaType": "application/vnd.docker.container.image.v1+json",
      "size": 1511,
      "digest": "sha256:183cb5fd54142948ad88cc986a217465cfe8786cfdd89b1ed1fc49825da413a7"
   },
   "layers": [
      {
         "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
         "size": 147,
         "digest": "sha256:c53fb220cbad89d19e819673ea42ffcb938319825ef36f1b96df37af3d4665f5"
      }
   ]
}
```

Reply:
huh… interesting… for some reason I managed to download this image (I do not remember if it was using `docker`, `podman` or `skopeo` lol.. :man_facepalming:). But that explain why it fails, mystery solved I'ld say.. (I also may have messed up when updating it… I need to proper check how I did it…)

Reply:
Thanks for the info @tianon. This explains it is not working but it should still return 404 instead 500. We will investigate internally depending on priority.